Source/ThirdParty/PlayCppSdkLibrary/Include/walletconnectcallback.h

```diff
@@ -1,65 +1,65 @@
-#pragma once
```

Comment:
what changes of this file, seems nothing changed, how about revert it back?

Reply:
ok